template.yml

```diff
@@ -123,6 +123,22 @@ Resources:
     Properties:
       CodeUri: lambda
       Handler: initializer.handler
+      Layers:
+        - !Ref SDKlayer
+      Policies:
+        - AWSLambdaBasicExecutionRole # Only logs
+        - Version: '2012-10-17' # allow Lambda actions
+          Statement:
+            - Effect: Allow
+              Action:
+                - lambda:GetAlias
+                - lambda:GetFunctionConfiguration
+              Resource: !Ref lambdaResource
+  publisher:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: lambda
+      Handler: publisher.handler
```

Comment:
I think the publisher does NOT need the `lambda:GetFunctionConfiguration` permission now (that call is performed by the initializer).

Reply:
`lambda:GetFunctionConfiguration` is used by the `waitUntilFunctionUpdated` SDK call to check whether the Lambda function has been updated, therefore it's still required here